docs/linux-packaging.md

```diff
@@ -180,9 +180,9 @@ script creates the binary packages:
 ```bash
 cd linux/${packageName}
 ~/ci-builder-scripts/bash/build-package \
-    --dists "xenial bionic" --arches "amd64 i386" \
+    --dists "focal bionic" --arches "amd64 i386" \
     --main-package-name "My great package" \
-    --supported-distros "xenial bionic focal" \
+    --supported-distros "bionic focal" \
```

Comment:
Should this file also mention the scripts/debian.sh script? (below/after the Launchpad section)

Reply:
Since this is documentation and it just gives an example of the command, I think we can omit groovy.